lib/src/Base/Algo/openturns/ApproximationAlgorithmImplementation.hxx

```diff
@@ -104,6 +104,9 @@ public:
   virtual Collection<Indices> getSelectionHistory(Collection<Point> & coefficientsHistory) const;
   virtual Point getErrorHistory() const;
 
+  /** isModelSelection accessor */
+  virtual Bool isModelSelection() const;
```

Comment:
the naming feels weird as its not an adjective; "isModelSelected" or just "modelSelection" ?

Reply:
- `isModelSelected` does not fit in my opinion, because the model itself is not selected: the coefficients in the model are.
- `modelSelection`: aren't we supposed to name a method which returns a `bool` as "is*"?